samples/snippets/create_lite_pubsub_export_subscription_example.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+# Copyright 2022 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This application demonstrates how to create a subscription that writes
+messages to a Pub/Sub topic. For more information, see documentation at
+https://cloud.google.com/pubsub/lite/docs/export-subscriptions.
+"""
+
+# [START pubsublite_create_pubsub_export_subscription]
+from google.api_core.exceptions import AlreadyExists
+from google.cloud.pubsub_v1 import PublisherClient
+from google.cloud.pubsublite import AdminClient, Subscription, ExportConfig
+from google.cloud.pubsublite.types import (
+    BacklogLocation,
+    CloudRegion,
+    CloudZone,
+    SubscriptionPath,
+    TopicPath,
+)
+
+
+def create_lite_pubsub_export_subscription(
+    project_number,
+    cloud_region="us-central1",
+    zone_id="a",
+    topic_id="my-topic-id",
+    subscription_id="my-subscription-id",
+    pubsub_topic_id="destination-topic-id",
+    regional=True,
+    target_location=BacklogLocation.END,
```

Comment:
Could we match this with what the Java sample has?

```java
    // Target location within the message backlog that the subscription should be initialized to.
    SeekTarget initialLocation = SeekTarget.of(BacklogLocation.BEGINNING);
```

Reply:
Done